src/index.js

```diff
@@ -0,0 +1,42 @@
+const inquirer = require("inquirer");
+const utilities = require("./utilities");
+const writeReadme = require("./readme");
+const { questions, installQuestions, testQuestions } = require("./questions");
+
+const start = async () => {
+  const readmeAnswers = await inquirer.prompt(questions);
+  const instructionsArray = [];
+  const testArray = [];
+
+  if (readmeAnswers.installationIncluded) {
```

Comment:
By implementing the destructured object as per the above comment, you can now target the `installationIncluded` key directly here:

`if (installationIncluded) { .... } `